packages/flutter/test/widgets/fade_transition_test.dart

```diff
@@ -12,8 +10,10 @@ void main() {
   testWidgets('FadeTransition', (WidgetTester tester) async {
     final DebugPrintCallback oldPrint = debugPrint;
     final List<String> log = <String>[];
-    debugPrint = (String message, { int wrapWidth }) {
-      log.add(message);
+    debugPrint = (String? message, { int? wrapWidth }) {
+      if (message != null) {
+        log.add(message);
```

Comment:
I think we should add a "null" message to the log. If that ever happens, the test should probably fail because we are logging something different then expected?